src/code/credits.asm

```diff
@@ -1213,7 +1221,7 @@ label_017_4C22:
     ret                                           ; $4C5A: $C9
 
 IncrementCreditsGameplaySubtype::
-IncrementCreditsGameplaySubtypeAndReturn::
+IncrementCreditsGameplaySubtypeAndreturn::
```

Comment:
Why this strange case change, here and on similar labels? It looks like you might have replaced all "Return" with "return" for some reason?

Reply:
Yup, clearly a mistake – I didn't even proof-read this PR myself 🤦 Fixed.